VTUL/VIVO_VT_Theme
# Refactor individual-properties so that publication renders on profile page

**Ensures that publications render on profile pages.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1669) (:star:)

# What does this Pull Request do? (:star:)
Refactors individual-properties.ftl to make publications render on profile pages. This file is also used to render the authors of publications and changes made there broke the profile page. This PR should fix that

# What's the changes? (:star:)
* Renders publication on profile page
<img width="1060" alt="screen shot 2019-01-23 at 1 09 57 pm" src="https://user-images.githubusercontent.com/1202435/51628318-b5b63380-1f12-11e9-9263-514bce5019c7.png">

* Renders authors on publication pages (with or without admin privileges)
<img width="1055" alt="screen shot 2019-01-23 at 1 10 26 pm" src="https://user-images.githubusercontent.com/1202435/51628404-ec8c4980-1f12-11e9-9231-d83aadf0fd27.png">

<img width="1029" alt="screen shot 2019-01-23 at 1 11 14 pm" src="https://user-images.githubusercontent.com/1202435/51628422-f31ac100-1f12-11e9-98c8-5b51cf2f1ab1.png">


# How should this be tested?
* Create a research publication
* Create a user (faculty)
* Add user as an author of the publication
* Make sure that user shows up as an author on the publication page
* Make sure that the publication shows up on the user's profile page

# Additional Notes:
* branch: `LIBTD-1669`

# Interested parties
@yinlinchen 

(:star:) Required fields
